api/python/provisioner/commands/sw_upgrade.py

```diff
@@ -137,7 +201,7 @@ def run(self, targets):  # noqa
 
                     minion_conf_changes = config_salt_minions()
```

Comment:
I've noticed that usage on `minion_conf_changes` is not clean enough: it is defined in a inner scope of try-catch but used later in an outer scope. It won't fail in current version but looks  not safe enough. Could you elaborate that more? Thanks

Reply:
This logic and these execution order moved from SW Update. I didn't nothing special for it. We can improve this but it will require additional validation.

Reply:
ok